### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by realvz**
Fixed markdown formatting.

*Issue #, if available:*

*Description of changes:*
Fixed markdown formatting 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/howto-k8s-alb/README.md [hunk 1]
```diff
@@ -33,11 +33,12 @@ Note: Only [deploy the ALB Ingress Controller](https://kubernetes-sigs.github.io
 
 Check the events of the ingress to see what has occur.
 
-    ```bash
+    ```
```
Comment (lavignes) [inline walkthroughs/howto-k8s-alb/README.md:36]: interest that putting the language broke the rendering here, but that is what it looks like...

**Comment by lavignes** [comment]
Thanks for fixing this!

File: walkthroughs/howto-k8s-alb/README.md (hunk 1 above)
Comment (dfezzie) [reply walkthroughs/howto-k8s-alb/README.md:36]: Looking at the rendered file in master it looks like there is still the issue. Could be an issue with line 28 which has a `.` after the backticks

Comment (lavignes) [reply walkthroughs/howto-k8s-alb/README.md:36]: Oh my bad @dfezzie

I hadn't looked at the rendered markdown on @realvz 's fork to verify. To be fair it actually looks mostly* correct with this change but yeah its still bugged. :)

**Pull request merged**